### DISCUSSION TITLE
Angular 18 Support and Angular 19 Upcoming Release

### DISCUSSION THREAD

**Discussion by sisindrymedagam** [vmware-clarity/ng-clarity#1536] [General]
Hi Clarity Design Team,

I wanted to discuss the support for Angular 18 in Clarity Design. As Angular 18 is already available, it would benefit many of us to have official support for it in Clarity. Ensuring compatibility with Angular 18 would allow developers to take advantage of the latest features and improvements while using Clarity for UI components.

Additionally, I noticed that [Angular 19](https://angular.dev/reference/releases#release-schedule) is already scheduled for release on the week of 2024-07-08. It would be great if the Clarity team could share any insights or plans regarding support for Angular 19. Knowing the timeline and planned updates would help developers better plan their projects and ensure a smooth transition when Angular 19 becomes available.

Note: Adding support for the latest Angular versions to the original Clarity framework would be a significant advantage. It would likely increase the usage and adoption of the Clarity UI framework, as developers prefer using libraries and frameworks that stay current with the latest technologies.

Looking forward to hearing your thoughts and any potential plans for supporting these upcoming versions of Angular.

Best regards,
@sisindrymedagam

**Comment by kevinbuhmann** [comment]
Angular 18 support is merged and will be included in our next release. I don't have an exact date, but it should be in the next couple weeks.

We do not currently have a timeline for adding Angular 19 support as it won't be released for more than two months.
